EdFi.Installer.AppCommon.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <RootNamespace>EdFi.Installer.AppCommon</RootNamespace>
+        <NoWarn>NU5110,NU5111</NoWarn>
+        <PackageReadmeFile>README.md</PackageReadmeFile>
+        <VersionPrefix>1.0.0</VersionPrefix>
+        <NuspecFile>EdFi.Installer.AppCommon.nuspec</NuspecFile>
+        <NuspecProperties>version=$(PackageVersion)</NuspecProperties>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <None Include="README.md" Pack="true" PackagePath="docs\" />
+    </ItemGroup>
+
+</Project>
```

Comment:
I created a csproj file that references the nuspec, thus allowing us to use `dotnet pack` (which requires a project file) instead of downloading and using `nuget.exe`.